MarsRover.Tests/StubOutput.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections.Generic;
+
+namespace MarsRover.Tests
+{
+    public class StubOutput : IOutput
+    {
+        public StubOutput()
+        {
+        }
+
+        public List<string> OutputList { get; private set; } = new List<string>();
+
+        public void Write(string value)
+        {
+            OutputList.Add(value);
+        }
+
+        public void WriteLine(string value)
+        {
+            OutputList.Add(value);
+        }
```

Comment:
How will you distinguish between `Write` and `WriteLine`? Presumably you care about the newline made by `WriteLine`